digitaltwin_client/src/dt_client_core.c

```diff
@@ -506,23 +496,7 @@ DT_CLIENT_CORE_HANDLE DT_ClientCoreCreate(DT_IOTHUB_BINDING* iotHubBinding, DT_L
 // themselves of the status of the registration.
 static void SetRegistrationCompleteInvokeCallbacks(DT_CLIENT_CORE* dtClientCore, DIGITALTWIN_CLIENT_RESULT dtInterfaceStatus)
```

Comment:
>SetRegistrationCompleteInvokeCallbacks [](start = 12, length = 38)

Please remove this function and just invoke DT_InterfaceList_RegistrationCompleteCallback() directly.